taxonomy-manager-rest/src/main/java/com/digirati/taxman/rest/taxonomy/ConceptResource.java

```diff
@@ -29,6 +31,12 @@
     @JsonLdFrame(input = "/jsonld/framing/concept.json")
     Response getConcept(@BeanParam ConceptPath params);
 
+    @GET
+    @Path("/by-label/{label}")
+    @Consumes({MediaType.APPLICATION_JSON})
```

Comment:
This method doesn't consume anything.

Reply:
This was copied from getConcept - happy to remove it from both

Reply:
Yep, should be removed from there too.